openedx/features/course_duration_limits/tests/test_access.py

```diff
@@ -1,23 +1,26 @@
 """Tests of openedx.features.course_duration_limits.access"""
 
-from datetime import datetime, timedelta
+from __future__ import absolute_import
+
 import itertools
+from datetime import datetime, timedelta
+
+import ddt
+from django.utils import timezone
+from pytz import UTC
 
 from course_modes.models import CourseMode
 from course_modes.tests.factories import CourseModeFactory
-from django.utils import timezone
 from courseware.models import DynamicUpgradeDeadlineConfiguration
 from openedx.core.djangoapps.schedules.tests.factories import ScheduleFactory
 from openedx.core.djangolib.testing.utils import CacheIsolationTestCase
 from openedx.features.course_duration_limits.access import (
     generate_course_expired_message,
-    get_user_course_expiration_date,
+    get_user_course_expiration_date
```

Comment:
@jmbowman: I'm surprised that `isort` doesn't leave the trailing comma. Is that something we can standardize on?

Reply:
It looks like there's an [include_trailing_comma](https://isort.readthedocs.io/en/latest/#multi-line-output-modes) setting for that.  Do you want to look through the isort settings and recommend changes in a PR?  Should probably make sure that the settings generated by `cookiecutter-django-app` are consistent.

Reply:
I'm not sure where these isort settings are. `~/.isort.cfg ` doesn't seem to exist in the docker instance.   The default value for `include_trailing_comma` is false... so, I'm still wonder if this is the exact cause.    Any ideas where I should look for this .isort.cfg file?

Reply:
It's in setup.cfg, which is one of the places isort looks for its configuration:

* https://github.com/edx/edx-platform/blob/d4cbcfd5d504b6a56cd37376d498bcd53bf0ac6d/setup.cfg#L36-L43
* https://github.com/timothycrosley/isort#configuring-isort

You don't need to do anything with that right now, although you're welcome to look into it if you're curious.  It's the default value for `include_trailing_comma` that we potentially have a problem with, this is where we would override it.